content/tutorials/channel-rewind.textile

```diff
@@ -26,6 +26,8 @@ Usually in applications that implement the Pub/Sub messaging pattern, a client i
 
 However, in some applications it is useful to have access also to messages that occurred in the past. Ably supports a "History API":/core-features/history whereby, for channels that are persisted, it is possible to query explicitly for historical messages on a channel within any specified time bounds.
 
+bq. You can now persist the last realtime message on a channel for 365 days. *State Persistence* allows you to instantly retrieve realtime messages sent in the past, helping with  long-term state synchronization. "Read how to enable State Persistence":https://www.ably.io/documentation/realtime/channels/channel-parameters/rewind#persist-last-message.
```

Comment:
Extra space between "helping with" and "long-term". Also link does not require full domain: `/realtime/channels/channel-parameters/rewind#persist-last-message` is enough.